src/applications/mhv-medical-records/util/pdfHelpers/allergies.js

```diff
@@ -10,9 +10,14 @@ export const generateAllergiesIntro = (records, lastUpdated) => {
   };
 };
 
-export const generateAllergyItem = record => {
+export const generateAllergyItem = (record, domain) => {
   const multipleReactions = record.reaction.length > 1;
 
+  const reactions =
+    domain === 'Allergy details'
+      ? record.reaction.map(react => `      •  ${react}\n`).join('')
```

Comment:
There might be a better way to handle lists, perhaps in the PDF builder itself. @mattwrightva are you able to offer guidance here.